### DISCUSSION TITLE
Only AGN, no host Galaxy

### DISCUSSION THREAD

**Discussion by muryelgp** [mboquien/cigale#80] [Q&A]
Hi, I'd like to use cigale to fit a source that is completely AGN dominate, the host galaxy is not even detected, it seems I cannot have only the skirtor2016, xray, redshifting SED modules, does it always requires a stellar component?

If I add a stellar component, but try to make the stellar component negligible, by doing fracAGN=0.999999999, it does not fit!

I get: 'No suitable model found for E2. It may be that models are older than the universe or that the χ² are very large.'

**Comment by Guang91** [comment]
Yes, you will need a stellar component and a fracAGN close to 1. 
As it says, could you check if your stellar age setting is older than the universe?

**Reply by muryelgp** [reply]
Thanks!

but, I already have stellar component and a fracAGN close to 1, still same error.
I have tried several ages, I always get the same thing.

**Comment by Guang91** [comment]
Could you share your pcigale.ini and data files?

**Reply by muryelgp** [reply]
[eRO2.txt](https://github.com/mboquien/cigale/files/14706857/eRO2.txt)
[pcigale.ini.txt](https://github.com/mboquien/cigale/files/14706859/pcigale.ini.txt)


I'v changed the from '.ini' to '.ini.txt' because '.ini' was not uploading here

**Reply by muryelgp** [reply]
On a side note. Fig 6 of your 2020 paper had a AGN only fit, is that not possible anymore?

**Comment by Guang91** [comment]
I think you will need a dust attenuation module and a dust emission module.
Although these are for galaxies, CIGALE needs them to normalize AGN
emission.

**Comment by muryelgp** [comment]
Makes sense, I will try. Thanks!

On a side note. Fig 6 of your 2020 paper had a AGN only fit, is that not possible anymore?
